Day-I-Social-Card/social-card/src/App.css

```diff
@@ -2,3 +2,13 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+
+.App {
+  height: 500px;
+  width: 45%;
+  margin: 1em auto;
+  border: 0.5px solid #000;
```

Comment:
`0.5px` visually does not make any difference to the border at all - it just reduces the padding, the smallest border size quantity is `1px`